toolchain/check/testdata/array/init_dependent_bound.carbon

```diff
@@ -101,7 +101,7 @@ fn H() { G(3); }
 // CHECK:STDOUT:   %array_type.loc9_24.2: type = array_type %int.convert_checked.loc9_23.2, constants.%i32 [symbolic = %array_type.loc9_24.2 (constants.%array_type)]
 // CHECK:STDOUT:   %require_complete: <witness> = require_complete_type %array_type.loc9_24.2 [symbolic = %require_complete (constants.%require_complete.7cb)]
 // CHECK:STDOUT:
-// CHECK:STDOUT:   fn(%N.patt.loc4_6.1: %i32) {
+// CHECK:STDOUT:   fn() {
```

Comment:
Is it expected that the param goes away entirely for `N:! i32`?

Reply:
Yeah, I assume when we support overloading, we'll have some way to uniquely identify individual overloads in the IR, and we'll use that in place of `@F` (or whatever).